portality/static/doaj/js/available_facetviews/admin.editorgroups.facetview.js

```diff
@@ -21,7 +21,7 @@ jQuery(document).ready(function($) {
             {'display':'Created Date','field':'created_date'},
             {'display':'Last Modified Date','field':'last_updated'},
             {'display':'Editor ID','field':'editor'},
```

Comment:
That would be to do with search rather than sort, but if we change it here we should change it there, too, since you'd probably be looking for the exact editor name string.